pkg/testing/integration_test.go

```diff
@@ -437,3 +456,39 @@ func pingAppAndWait(ctx context.Context, a app.DatabaseApp) error {
 	})
 	return err
 }
+
+// getServiceAccount return ServiceAccount object
+func getServiceAccount(namespace string) *v1.ServiceAccount {
+	return &v1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      namespace + "-sa",
+			Namespace: namespace,
+		},
+	}
+}
+
+// getClusterRoleBinding returns ClusterRoleBinding object which grants cluster-admin permission to
+// given ServiceAccount
+func getClusterRoleBinding(sa *v1.ServiceAccount) *rbacv1.ClusterRoleBinding {
+	return &rbacv1.ClusterRoleBinding{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ClusterRoleBinding",
+			APIVersion: "rbac.authorization.k8s.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: sa.Namespace + "-cluster-admin",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "default",
```

Comment:
Instead of binding roles to `default` SA, we should create a new SA and add the role to it